### PR TITLE
Feature/docker integration

### PR DESCRIPTION
**Pull request by k911**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -42,3 +51,10 @@ deploy:
     on:
       branch: develop
       node: '8'
+  - provider: script
+    skip_cleanup: true
+    script: bash ./deploy/docker-app-push.sh
+    on:
+      tags: true
+      condition: "$DOCKER_APP_PUSH = 1"
+
```
Comment (FieryCod) [inline .travis.yml]: trailing newline

**Comment by FieryCod** [comment]
Seems legit very good job congrats :)

File: .travis.yml (hunk 1 above)
Comment (k911) [reply .travis.yml]: Have you seen `.editorconfig`? AFAIK It's good practice, to easily insert something in the future at the EOF :) so it's should stay

Comment (k911) [reply .travis.yml]: Unless it's double new line..

**Pull request merged**